zeppelin-web/src/components/login/login.controller.js

```diff
@@ -39,5 +39,15 @@ angular.module('zeppelinWebApp').controller('LoginCtrl',
       });
 
     };
+
+    $rootScope.$on('initLoginValues', function() {
```

Comment:
$scope.$on, not $rootScope.$on

Can we also move the content of initValues function inside this block since this is the only one calling that function?